db/migrate/20150514072626_add_type_to_config_template.rb

```diff
@@ -0,0 +1,6 @@
+class AddTypeToConfigTemplate < ActiveRecord::Migration
+  def change
+    add_column :config_templates, :type, :string, :default => 'ConfigTemplate'
```

Comment:
Yes, at this moment we want all records in that table with blank type to become ConfigTemplate, in later migration we migrate ConfigTemplate to ProvisioningTemplate. Maybe we could change the default type there, but we should never get a record without type explicitly set.